official/vision/beta/projects/yolo/dataloaders/yolo_input.py

```diff
@@ -0,0 +1,511 @@
+        assignment. Anchor free will perform worse than Anchor based, but only 
+        slightly.
+      max_num_instances: `int` for the number of boxes to compute loss on.
+      area_thresh: `float` for the minimum area of a box to allow to pass 
+        through for optimization.
+      aug_rand_hue: `float` indicating the maximum scaling value for 
+        hue. saturation will be scaled between 1 - value and 1 + value.
+      aug_rand_saturation: `float` indicating the maximum scaling value for 
+        saturation. saturation will be scaled between 1/value and value.
+      aug_rand_brightness: `float` indicating the maximum scaling value for 
+        brightness. brightness will be scaled between 1/value and value.
+      letter_box: `boolean` indicating whether upon start of the datapipeline 
+        regardless of the preprocessing ops that are used, the aspect ratio of 
+        the images should be preserved.  
+      random_pad: `bool` indiccating wether to use padding to apply random 
+        translation true for darknet yolo false for scaled yolo.
+      random_flip: `boolean` indicating whether or not to randomly flip the 
+        image horizontally. 
+      jitter: `float` for the maximum change in aspect ratio expected in 
+        each preprocessing step.
+      aug_scale_min: `float` indicating the minimum scaling value for image 
+        scale jitter. 
+      aug_scale_max: `float` indicating the maximum scaling value for image 
+        scale jitter.
+      aug_rand_translate: `float` ranging from 0 to 1 indicating the maximum 
+        amount to randomly translate an image.
+      aug_rand_perspective: `float` ranging from 0.000 to 0.001 indicating 
+        how much to prespective warp the image.
+      aug_rand_angle: `float` indicating the maximum angle value for 
+        angle. angle will be changes between 0 and value.
+      anchor_t: `float` indicating the threshold over which an anchor will be 
+        considered for prediction, at zero, all the anchors will be used and at
+        1.0 only the best will be used. for anchor thresholds larger than 1.0 
+        we stop using the IOU for anchor comparison and resort directly to 
+        comparing the width and height, this is used for the scaled models.  
+      scale_xy: dictionary `float` values inidcating how far each pixel can see 
+        outside of its containment of 1.0. a value of 1.2 indicates there is a 
+        20% extended radius around each pixel that this specific pixel can 
+        predict values for a center at. the center can range from 0 - value/2 
+        to 1 + value/2, this value is set in the yolo filter, and resused here. 
+        there should be one value for scale_xy for each level from min_level to 
+        max_level.
+      best_match_only: `boolean` indicating how boxes are selected for 
+        optimization.
+      coco91to80: `bool` for wether to convert coco91 to coco80 to minimize 
```

Comment:
Uhmm, I feel this can be done as part of the decoder? I feel it is better to keep this parser coco-free.